starter/package.json

```diff
@@ -112,51 +112,52 @@
     ]
   },
   "dependencies": {
-    "@angular/animations": "~7.2.2",
-    "@angular/cdk": "~7.3.0",
-    "@angular/cli": "~7.3.0",
-    "@angular/common": "~7.2.2",
-    "@angular/compiler": "~7.2.2",
-    "@angular/compiler-cli": "~7.2.2",
-    "@angular/core": "~7.2.2",
-    "@angular/flex-layout": "7.0.0-beta.24",
-    "@angular/forms": "~7.2.2",
-    "@angular/material": "~7.3.0",
-    "@angular/platform-browser": "~7.2.2",
-    "@angular/platform-browser-dynamic": "~7.2.2",
-    "@angular/platform-server": "~7.2.2",
-    "@angular/router": "~7.2.2",
-    "@nationalbankbelgium/code-style": "^1.2.0",
-    "@nationalbankbelgium/stark-core": "latest",
-    "@nationalbankbelgium/stark-ui": "latest",
+    "@angular/animations": "^8.2.0",
+    "@angular/cdk": "^8.2.0",
+    "@angular/cli": "^8.3.0",
+    "@angular/common": "^8.2.0",
+    "@angular/compiler": "^8.2.0",
+    "@angular/compiler-cli": "^8.2.0",
+    "@angular/core": "^8.2.0",
+    "@angular/flex-layout": "^8.0.0-beta.25",
+    "@angular/forms": "^8.2.0",
+    "@angular/material": "^8.2.0",
+    "@angular/platform-browser": "^8.2.0",
+    "@angular/platform-browser-dynamic": "^8.2.0",
+    "@angular/platform-server": "^8.2.0",
+    "@angular/router": "^8.2.0",
+    "@nationalbankbelgium/code-style": "^1.4.0",
+    "@nationalbankbelgium/stark-core": "file:../dist/packages-dist/stark-core/nationalbankbelgium-stark-core-10.2.0-a92e5a34.tgz",
+    "@nationalbankbelgium/stark-ui": "file:../dist/packages-dist/stark-ui/nationalbankbelgium-stark-ui-10.2.0-a92e5a34.tgz",
     "@uirouter/visualizer": "~7.2.1",
     "core-js": "~3.8.0",
     "eligrey-classlist-js-polyfill": "~1.2.20180112",
-    "event-source-polyfill": "~1.0.5",
+    "event-source-polyfill": "~1.0.22",
     "hammerjs": "^2.0.8",
     "http-server": "~0.12.0",
     "ngrx-store-freeze": "~0.2.4",
     "ngrx-store-logger": "~0.2.2",
     "reflect-metadata": "~0.1.13",
-    "rxjs": "^6.0.0",
-    "zone.js": "~0.10.2"
+    "rxjs": "^6.6.6",
+    "zone.js": "~0.10.3"
   },
   "devDependencies": {
     "@compodoc/compodoc": "~1.1.11",
-    "@nationalbankbelgium/stark-build": "latest",
-    "@nationalbankbelgium/stark-testing": "latest",
-    "@types/core-js": "~2.5.0",
-    "@types/hammerjs": "~2.0.36",
-    "@types/node": "~10.17.13",
-    "@types/webpack-env": "~1.15.0",
-    "cross-env": "~7.0.0",
+    "@nationalbankbelgium/stark-build": "file:../dist/packages-dist/stark-build/nationalbankbelgium-stark-build-10.2.0-a92e5a34.tgz",
+    "@nationalbankbelgium/stark-testing": "file:../dist/packages-dist/stark-testing/nationalbankbelgium-stark-testing-10.2.0-a92e5a34.tgz",
+    "@types/core-js": "~2.5.4",
+    "@types/hammerjs": "~2.0.39",
+    "@types/node": "~10.17.54",
+    "@types/webpack-env": "~1.16.0",
+    "browserstack-local": "~1.4.8",
```

Comment:
Is it required to add "browserstack-local" ?
As we don't have any e2e tests for the starter, could you maybe remove it ? 😊

Reply:
Yeah true, but I thought, it would be (in the future) a push to make the projects use it. At least a bit :-).
It's removed.